# ouster

  fx: 1419.488223
  fy: 1419.783836
  cx: 2091.72389
  cy: 1231.71474
  k1: 0.4765817545
  k2: 0.0267510319
  p1: 1.495544387e-07
  p2: -2.581193063e-07
  k3: 8.833337814e-05
  k4: 0.8074516265
  k5: 0.1103335164
  k6: 0.001695886702

# 1419.488223, 1419.783836, 2091.72389, 1231.71474, 
# 0.4765817545, 0.0267510319, 1.495544387e-07, -2.581193063e-07, 
# 8.833337814e-05, 0.8074516265, 0.1103335164, 0.001695886702

# Calibration target parameters
  marker_size: 0.20 # ArUco marker size (our test data uses 0.16m; adjust to match your marker size)
  delta_width_qr_center: 0.55 # Half the distance between the centers of two markers in the horizontal direction 
  delta_height_qr_center: 0.35 # Half the distance between the centers of two markers in the vertical direction
  delta_width_circles: 0.5 # Distance between the centers of two circles in the horizontal direction
  delta_height_circles: 0.4 # Distance between the centers of two circles in the vertical direction
  circle_radius: 0.12 # Radius of the circle 

# Distance filter
  x_min: 2.2
  x_max: 2.6
  y_min: -1.5
  y_max: 2.0
  z_min: -0.8 # avia and mid360: -0.5 ouster: -0.8
  z_max: 2.0

# Input
  lidar_topic: "/ouster/points" 
  bag_path: "$(find fast_calib)/calib_data/11.bag"
  image_path: "$(find fast_calib)/calib_data/our_image.jpg"

# Output
  output_path: "$(find fast_calib)/output"

  use_custom_plane: true
  custom_plane_path: "/home/ali/workspaces/calib_ws/src/FAST-Calib/calib_data/our_checkboard.pcd"